x-pack/plugins/ml/public/application/components/influencers_list/influencers_list.tsx

```diff
@@ -40,13 +62,12 @@ function getTooltipContent(maxScoreLabel, totalScoreLabel) {
   );
 }
 
-function Influencer({ influencerFieldName, influencerFilter, valueData }) {
-  const maxScorePrecise = valueData.maxAnomalyScore;
-  const maxScore = parseInt(maxScorePrecise);
-  const maxScoreLabel = maxScore !== 0 ? maxScore : '< 1';
+const Influencer: FC<InfluencerProps> = ({ influencerFieldName, influencerFilter, valueData }) => {
+  const maxScore = Math.floor(valueData.maxAnomalyScore);
+  const maxScoreLabel = maxScore !== 0 ? `${maxScore}` : '< 1';
```

Comment:
nit: we have several places in code with this logic. maybe it's worth introducing a tiny utility function `getScoreAsString` that takes care of rounding the value and this `< 1` label.

Reply:
Switched to existing getFormattedSeverityScore function in https://github.com/elastic/kibana/pull/90282/commits/d02a96a8e9fa3515dcbe02f80d1e476d2d430133